atom/common/api/atom_bindings.cc

```diff
@@ -51,35 +51,44 @@ AtomBindings::~AtomBindings() {
   uv_close(reinterpret_cast<uv_handle_t*>(&call_next_tick_async_), nullptr);
 }
 
+// static
+void AtomBindings::BindProcess(v8::Isolate* isolate,
+                               mate::Dictionary* process,
+                               base::ProcessMetrics* metrics) {
+  // These bindings are shared between sandboxed & unsandboxed renderers
+  process->SetMethod("crash", &Crash);
+  process->SetMethod("hang", &Hang);
+  process->SetMethod("log", &Log);
```

Comment:
"log" was not available in sandbox renderers originally. Is it intentional to add it and if so should it also be added to process.md docs?

Reply:
Log is not documented even in non-sandboxed renderer